module.exports = {
    invalidLogin: 'Invalid Login.',
    emailInUse: 'Email is already in use by some other person.',
    userDoNotExist: 'User do not exist',
    adminDoNotExist: `Can't find this admin`,
    invalidPassword: 'Invalid Password',
    userIsBlocked: 'Your ID is blocked. Please contact Admin!',
    success: 'success',
    passwordNotMatched: 'Passwords dont match',
    insufficientData: 'Not enough data provided',
    registerIssue: 'Could not register the user',
    loginIssue: 'Could not log the user in',
    logoutIssue: 'Could not log the user out',
    noUsers: 'No user exists',
    updationFailed: 'Updation failed',
    noDeletion: 'Could not perform the deletion',
    profileFetchingErr: 'Error retrieving user profile',
    invalidRole: 'This role already exists',
    roleCreationIssue: 'Failed to create this role',
    whileUpdatingRole: 'Role does not exist',
    roleUpdationIssue: 'Failed to update this role',
    roleDeletionIssue: 'Failed to delete this role',
    userDeletionIssue: 'Failed to delete this user',
    forbidden: 'Access denied!',
    invalidProject: 'Project already exists',
    projectCreationIssue: 'Failed to create this project',
    projectUpdationIssue: 'Failed to update this project',
    projectDeletionIssue: 'Failed to delete this project',
    projectFetchingErr: 'Error retrieving the project',
    noProjects: 'No projects exist',
    projectsFetchingErr: 'Error retrieving all projects',
    projectDoNotExist: 'Project do not exist',
    invalidTask: 'Task already exists',
    taskCreationIssue: 'Failed to create this task',
    taskUpdationIssue: 'Failed to update this task',
    taskDeletionIssue: 'Failed to delete this task',
    taskFetchingErr: 'Error retrieving the task',
    taskDoNotExist: 'Task do not exist',
    noTasks: 'No tasks exist',
    noProjectAssigned: 'This user has no assigned project'
};